### DISCUSSION TITLE
Android support wirh Vulkan

### DISCUSSION THREAD

**Discussion by falhumai96** [LukasBanana/LLGL#140] [Q&A] [answered]
Hello,

Can you tell me about the Vulkan support for Android? Why does it have a 🚧 emoji (or what does it mean)?

**Comment by LukasBanana** [comment, marked as answer]
The "under construction" tag should probably apply to all platforms for the Vulkan backend, but on mobile devices any issues are much more noticeable. I tested the HelloGame example with Vulkan in the Android emulator where it worked just fine, but it works not at all on my Android tablet. I don't have a high end Android device, but it's an indicator that the Vulkan backend is not ready for Android yet nonetheless.

**Reply by falhumai96** [reply]
Understandable. Thank you so much!
